src/main/scala/dpla/ingestion3/entries/ingest/JsonlEntry.scala

```diff
@@ -35,15 +39,34 @@ object JsonlEntry {
       .setAppName("jsonl")
       .setMaster("local[*]")
 
-    val spark = SparkSession.builder().config(sparkConf).getOrCreate()
+
+    executeJsonL(sparkConf, inputName, outputName, Utils.createLogger("jsonl", ""))
+  }
+
+  /**
+    * Generate JSON-L files from AVRO file
+    * @param sparkConf Spark configuration
+    * @param dataIn Data to transform into JSON-L
+    * @param dataOut Location to save JSON-L
+    */
+  def executeJsonL(sparkConf: SparkConf, dataIn: String, dataOut: String, logger: Logger) = {
```

Comment:
I think eventually we should break out these `execute` methods to their own traits or classes and have the `Entries` just have the single main method. 

Then that trait or abstract class can have methods to inspect the results of the run and make decisions about it. Would also be useful for integrating into something like Combine.